custom-graders/v2/README.md

```diff
@@ -0,0 +1,37 @@
+# New V2 Format Custom grader samples
+
+### What is new in V2?
+Please refer to this [doc](https://docs.google.com/UPDATE_THE_LINK) to know more about what is available in autograder v2.
+
+### Demo Java Grader
+
+DemoJavaGrader is one custom grader sample demonstrates what mainly changed in autograder v2.
+
+Graders now need to produce ``feedback.json`` in ``/shared`` folder in the same format as before, which is : 
+```sh
+{"fractionalScore": 0.2, "feedback": "You failed"}
+```
+
+### Demo Java Grader With Rich Feedback
+
+In V2, we now support rich feedback. DemoJavaGraderWithRichFeedback demonstrates how to provide rich feedback. This is an optional feature. If graders decides to provide rich feedback, then we only show rich feedback to the learner.
+
+Custom grader can indicate rich feedback, by providing the optional field "feedbackType" in ``feedback.json``. This can take either "HTML" or "TXT" as values. It is also important for the grader to provide the rich feedback in ``/shared/`` folder, ``htmlFeedback.html`` for "HTML" and ``txtFeedback.txt`` for "TXT" respectively.
+
+```sh
+{"fractionalScore": 0.2, "feedback": "You failed", "feedbackType": "HTML"}
+```
+
+### Demo Java Grader With Network
+
+In V2, coursera can access certain predefined domains, from the the grader. This example demonstrates downloading coursera.org page into ``/shared/htmlFeedback.html`` and provide this as rich feedback. 
+
```

Comment:
I recommend placing Python grader at the top, perhaps below Demo Java Grader. More partners use Python compared to Java, so I think it might be something closer to what they are looking for.

Reply:
Sounds good. Will move.